### PR TITLE
Add usage example

### PR DESCRIPTION
**Pull request by lucianonooijen**
* Added clarification that you need a folder with types only, as I got `Error: No decoder named: tDecoder found!` errors otherwise. 
* Added example for what I mentioned above in the repo

I hope this will lower the bar a bit for people willing to try Aegis out. 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,28 @@ Here are some steps to get you going:
...
+After running this command, you will be greeted with an `example.ts` file in the root of this repository, containing the generated decoders.
+
+To use the decoder, you must create a guard using the `decoders` package as shown below:
+
+```ts
+import { oneTypeDecoder } from "./example";
+import { guard } from "decoders";
+
+const checker = guard(oneTypeDecoder)
+const data = { /* your data following the oneTypeDecoder definition */ }
```
Comment (LcTwisk) [inline README.md]:
```suggestion
const checker = guard(oneTypeDecoder);
const data = { /* your data following the oneTypeDecoder definition */ };
```
🤓

**Review by LcTwisk — APPROVED** [review]
Hi @lucianonooijen, thanks for your nice contribution! 
This will definitely make things easier 👍

File: example/onetype.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export default interface OneType {
+    someProp: number;
```
Comment (LcTwisk) [inline example/onetype.ts:7]: Would maybe be even more educative to include nested types in the example? By changing this to: 
`someProp: AnotherType;`
WDYT?

Comment (lucianonooijen) [reply example/onetype.ts:7]: Would be a great addition! Added it

**Comment by lucianonooijen** [comment]
Just saw I also had to update the `tsconfig.json` file to tell the compiler not to include the examples in the build (otherwise `./lib/index.js` would also break as it would then require `./lib/src/index.js`, now it's fine)

**Pull request merged**